public/js/lesson-lists.js

```diff
@@ -0,0 +1,50 @@
+$(function() {
+  initializeLocalStorage();
+  let modulePreference = getModulePreference();
+  activateButton(modulePreference);
+  showOrHideLessonContainer(modulePreference, 0);
+
+  $('.module-selector').on('click', function(event) {
+    let previousModulePreference = getModulePreference();
+    deactivateButton(previousModulePreference);
+
+    let moduleSelection = $(event.target).data('module');
+    setModulePreference(moduleSelection);
+    activateButton(moduleSelection);
+    showOrHideLessonContainer(moduleSelection, 100);
+  });
+
+  function initializeLocalStorage() {
```

Comment:
I think this has the potential to be fragile if the `fe-curriculum-lessons` key is there, but the object inside is missing. Interested in a better, more robust solution for this.

Reply:
What does the data structure for `fe-curriculum-lessons` look like right now? Is it just:

```js
fe-curriculum-lessons = {
    showModule: null
}
```

If so, I feel like we should just simplify this to a single key-value pair rather than stringifying an object. We can make it more complex when necessary. e.g. `turing-modulePref: 2`

Don't really know right now I'm drunk just tryna help.